lib/text/vtt_text_parser.js

```diff
@@ -463,14 +465,25 @@ shaka.text.VttTextParser = class {
     payload = VttTextParser.replaceKaraokeStylePayload_(payload);
     payload = VttTextParser.replaceVoiceStylePayload_(payload);
     const xmlPayload = '<span>' + payload + '</span>';
-    const element = shaka.util.XmlUtils.parseXmlString(xmlPayload, 'span');
+    let element;
+    try {
+      element = shaka.util.TXml.parseXmlString(xmlPayload, 'span');
+    } catch (e) {
+      shaka.log.warning('cue parse fail: ', e);
+      element = {
```

Comment:
We don't want an element here because we already have a fallback, so this is not necessary